script/e2e-test.sh

```diff
@@ -396,7 +398,7 @@ kubectl create serviceaccount kubeapps-edit -n kubeapps
 # default). See https://github.com/vmware-tanzu/kubeapps/issues/4435
 kubectl create rolebinding kubeapps-edit -n kubeapps --clusterrole=edit --serviceaccount kubeapps:kubeapps-edit
 kubectl create rolebinding kubeapps-edit -n default --clusterrole=edit --serviceaccount kubeapps:kubeapps-edit
-kubectl create rolebinding kubeapps-repositories-read -n kubeapps --clusterrole kubeapps:kubeapps:apprepositories-read --serviceaccount kubeapps:kubeapps-edit
+kubectl create clusterrolebinding kubeapps-repositories-read --clusterrole kubeapps:kubeapps:apprepositories-read --serviceaccount kubeapps:kubeapps-edit
```

Comment:
Changing to clusterrolebinding instead for allowing listing repos when using `namespace=""`
Issue tracked at https://github.com/vmware-tanzu/kubeapps/issues/5215